tests/testthat/test-session.property.R

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) 2015-present, Facebook, Inc.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+
+context("session.property")
+
+source('utilities.R')
+
+query_session_property <- function(conn, property) {
+  properties <- dbGetQuery(conn, 'SHOW SESSION')
+  properties[properties$Name == property, ]$Value
```

Comment:
Please use list[['item']] style of indexing for accessing list items.

You can also do:

    return(properties[properties[['Name']] == property, 'Value', drop=TRUE])